hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/serialization/SerializableByConvention.java

```diff
@@ -16,16 +16,14 @@
 
 package com.hazelcast.internal.serialization;
 
-import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
-
 import java.lang.annotation.Retention;
 import java.lang.annotation.Target;
 
 import static java.lang.annotation.ElementType.TYPE;
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * Indicates that the annotated class cannot be converted to {@link IdentifiedDataSerializable} due to other conventions.
+ * Indicates that the annotated class cannot be converted to {@code IdentifiedDataSerializable} due to other conventions.
```

Comment:
Because it needs to be put on production classes. Personally I dislike such annotations.

Reply:
The only reason for this annotation is for finding the classes for the test case. It can be solved in other ways, e.g. the file names can be listed in a test file and we avoid any change in production for test purposes.

Reply:
Changing such logic is out of the scope of this PR. This is existing functionality that I needed to move to TPC.